packages/core/src/workflows/workflow.ts

```diff
@@ -39,6 +40,9 @@ export class Workflow<TSteps extends Step<any, any, any>[] = any, TTriggerSchema
   #actor: ReturnType<typeof createActor<ReturnType<typeof this.initializeMachine>>> | null = null;
   #runId: string;
   #retryConfig?: RetryConfig;
+  #engine?: MastraEngine;
+  #connectionId = `WORKFLOWS`;
+  #entityName = `__workflows__`;
   #telemetry?: Telemetry;
 
```

Comment:
Do we still want  telemetry, logger as workflow constructor properties since we now inject them as dependencies from mastra